server/utils/index.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+const COMBINED_UID_JOINSTR = ', ';
+
+const escapeUid = (uid) => uid.replace(/,/g, '').replace(COMBINED_UID_JOINSTR, '');
```

Comment:
This function isn't used. What is it for?

Reply:
I had used it here: https://github.com/trieb-work/strapi-plugin-config-sync/blob/feat-combined-uids/server/config/type.js#L17
And also in line 19.

Seems like I accidentally removed it. It was there to prevent the Problem you described in the above comment (if UID includes a `,`)

Will bring it back in the next commit

Reply:
Okay, just realized that I need to escape the values and not the keys. So it probably will become more difficult and impractical to escape the values, because the escaped values could not be used for the where filter anymore.

Reply:
But what's wrong with the implementation as it is right now?
It seemed to work on my end.

Reply:
Jep, it works as long as the join key does not occur in one off the variables itself. For example If you decide to use a page title as part of the UID and if the page title includes the `, ` then it will not work to split and assign the filename values to the keys.

I think I might have a solution. Will commit it later :)